rs_cleaner.py

```diff
@@ -0,0 +1,24 @@
+class RSCleaner:
```

Comment:
let's name this `PostCleaner` or something (and change the filename too). actually I think I'd rather see this change in a different PR because it's not actually complete and isn't directly related to the ticket

Reply:
^ nevermind, we'll keep it